apps/fluent-tester/src/FluentTester/theme/CustomThemes.ts

```diff
@@ -21,8 +21,10 @@ export const lightnessOptions = [
   { label: 'Dark', value: 'dark' },
 ];
 
+const defaultAsThemeName = ['android', 'ios', 'macos', 'web'].includes(Platform.OS as string);
```

Comment:
Might be easier to inverse this and just say "if win32, then Office?" :D

Reply:
Yep! Making this change.